config/settings.yaml.example

```diff
@@ -127,3 +127,18 @@
 #
 # :dynflow:
 #   :pool_size: 5
+
+#
```

Comment:
What do you think about creating a nested set of cache options to namespace them? Something akin to:

```
:cache_store:
  :type: redis
  :namespace: foreman
  :compress: true
  :hosts:
    - redis://localhost:8479
```

If some of these options are quite redis specific, I think that could still live nested under something like `redis_options`.

Reply:
Actually, having Redis available opens up many new possibilities. I am thinking about exploring queuing for hooks or facts cache for faster updates of facts. But these instances should be probably separate, therefore this nicely fits this if I want to add new key `:facts_cache_store` in the future. Can you rename it to `:rails_cache_store` instead to differentiate it clearly?

Reply:
I had such a structure in the initial implementation and changed it because I thought we should probably just specify a single redis instance. But I agree, we should use a separate configuration.

Reply:
I would add, that we are thinking about abusing redis for few more purposes than just caching and it would be nice to share the configuration of redis throughout all of its uses. Would it make sense?

Reply:
I‘m fine with both.